ovisbot/extensions/utils/utils.py

```diff
@@ -1,13 +1,26 @@
 import logging
+from ovisbot.helpers import chunkify
 import struct
 import string
 
+
 from Crypto.Util.number import long_to_bytes, bytes_to_long
 from discord.ext import commands
 
 logger = logging.getLogger(__name__)
 
 
+def rotn_helper(offset, text):
```

Comment:
This change seems unrelated to the goal of this PR - can you please rebase so it goes away and only the notebook related changes are left? 🙏

Reply:
should be OK now :)